apis/projectcontour/v1/httpproxy.go

```diff
@@ -152,6 +152,9 @@ type Route struct {
 	//
 	// +kubebuilder:validation:Optional
 	PathRewrite *PathRewritePolicy `json:"pathRewritePolicy,omitempty"`
+	// The policy for managing headers during proxying
+	// +optional
+	HeaderPolicy *HeaderPolicy `json:"headerPolicy,omitempty"`
```

Comment:
Sorry for bringing this up late. This field needs to be called `requestheaderpolicy` so we can add `responseHeaderPolicy` later.

Reply:
I suggest we call this `RequestHeadersRewrite` to be consistent with `PathRewrite`.

Reply:
@jpeach but this isn't just a rewrite. In the future, we could add & remove headers.